lambda/index.js

```diff
@@ -0,0 +1,59 @@
+const https = require('https')
+const URL = require('url').URL
+
+exports.handler = function (event, context) {
+  console.debug('Event: ' + JSON.stringify(event, null, 2))
+
+  const webhookUrl = new URL(process.env.SLACK_WEBHOOK_URL)
+  const requestPromises = []
+  for (const record of event.Records) {
+    requestPromises.push(_sendSlackMessage(record.Sns.Message, webhookUrl))
+  }
+
+  Promise.all(requestPromises).then(() => {
+    console.info('All messages send successfully.')
+  }).catch(e => {
+    console.error(e)
+    context.fail(e)
+  })
+}
+
+function _sendSlackMessage (messageText, webhookUrl) {
+  const data = JSON.stringify({
+    type: 'mrkdwn',
+    text: `*[${process.env.APP_NAME.toUpperCase()}]* ${messageText}`
+  })
```

Comment:
```suggestion
// https://api.slack.com/reference/surfaces/formatting#escaping
const ampRegex = /&/g
const ltRegex = /</g
const gtRegex = />/g
function escapeForSlack (string) {
  return string.replace(ampRegex, '&amp;').replace(ltRegex, '&lt;').replace(gtRegex, '&gt;')
}

function _sendSlackMessage (messageText, webhookUrl) {
  const data = JSON.stringify({
    type: 'mrkdwn',
    text: `*[${process.env.APP_NAME.toUpperCase()}]* ${escapeForSlack(messageText)}`
  })
```

Reply:
I'm sure there's more we can do to pretty up the message, but this change will at least keep us from breaking our formatting.